nosekatreport/plugin.py

```diff
@@ -628,11 +690,8 @@ def wait(cls, seconds, message=None):
     @classmethod
     def progress(cls, message):
         """Add progress messages to the step."""
-        # _state.store.add_progress(message)
-        _state.store.add_progress(message)
```

Comment:
Uncommented for now. Might want to add a command line parameter to determine whether to record the progress steps or not.